site2/website-next/docusaurus.config.js

```diff
@@ -143,54 +142,6 @@ module.exports = {
           position: "right",
           label: "Docs",
         },
-        {
-          label: "REST APIs",
-          position: "right",
-          items: [
-            {
-              label: "Admin REST API",
-              to: "/admin-rest-api",
-            },
-            {
-              label: "Functions",
-              to: "/functions-rest-api",
-            },
-            {
-              label: "Sources",
-              to: "/source-rest-api",
-            },
-            {
-              label: "Sinks",
-              to: "/sink-rest-api",
-            },
-            {
-              label: "Packages",
-              to: "/packages-rest-api",
-            },
-          ],
-        },
-        {
-          label: "CLI",
-          position: "right",
-          items: [
-            {
-              label: "Pulsar Admin",
-              to: "/pulsar-admin-cli",
-            },
-            {
-              label: "Pulsar Client",
-              to: "/pulsar-client-cli",
-            },
-            {
-              label: "Pulsar Perf",
-              to: "/pulsar-perf-cli",
-            },
-            {
-              label: "Pulsar",
-              to: "/pulsar-cli",
```

Comment:
In our new IA, these belong to the "Reference" chapter, so we can remove these, correct? @D-2-Ed @DaveDuggins

Reply:
Talked w/ @ @D-2-Ed, we can:
- put the CLI link to "Reference > [Pulsar CLI tools](https://pulsar.apache.org/docs/en/reference-cli-tools) chapter"
- put the REST API link to "Reference > REST API chapter" (➡️we need to create this new RESTAPI.md). We'd better create that file before switch over or else users can't find the REST ARPI doc entrance.

Reply:
I'm a little confused as to what's being asked here, but as a general rule, we don't want to change anything in the current docs that results in broken links, or links to pages that haven't been created yet. We will deploy those changes after the launch, section by section, throughout the year.